tests/_utils/test-lazy-loading-rules.js

```diff
@@ -56,8 +56,8 @@ addHook(
  * Everything related to loading any ESLint modules should be in this IIFE
  */
 (async () => {
-    const { ESLint } = require("../..");
-    const eslint = new ESLint({ cwd });
+    const { LegacyESLint } = require("../../lib/unsupported-api");
+    const eslint = new LegacyESLint({ cwd });
```

Comment:
This file is used only in the `legacy-eslint.js` tests so I updated it to use the LegacyESLint class, though I'm not sure why we didn't have an equivalent test for the FlatESLint (now ESLint) class.